### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by taowaren**
Could you please allow me to advertise your mod to MCBBS?
MCBBS is the largest Minecraft forum in China.
I'll mark you as the author.
I'll send you a link after I finish the post.
I'll show them your curseforge page and let them download the mod there.
Hope for your reply!

### CODE REVIEW

**Comment by Prospector** [comment]
Sure, that is fine with me. Thanks for finishing the translation!

File: src/main/resources/assets/traverse/lang/zh_cn.json [hunk 1]
```diff
@@ -43,4 +54,6 @@
   "block.traverse.fir_fence_gate": "冷杉木栅栏门",
   "block.traverse.fir_door": "冷杉木门",
   "block.traverse.fir_trapdoor": "冷杉木活板门"
```
Comment (coderbot16) [inline src/main/resources/assets/traverse/lang/zh_cn.json:56]: It looks like you're missing a comma at the end of this line.

**Review by coderbot16 — CHANGES_REQUESTED** [review]
Looks good to me, just noticed a JSON syntax error.

File: src/main/resources/assets/traverse/lang/zh_cn.json (hunk 1 above)
Comment (taowaren) [reply src/main/resources/assets/traverse/lang/zh_cn.json:56]: thank

**Comment by taowaren** [comment]
https://www.mcbbs.net/thread-1019142-1-1.html

**Comment by Prospector** [comment]
Thanks for showing me! I noticed that this link here is wrong
image
It links to the old Forge version instead of the right link which should be https://www.curseforge.com/minecraft/mc-mods/traverse/files

**Pull request merged**